WordPress/src/main/java/org/wordpress/android/util/AnalyticsUtils.java

```diff
@@ -46,6 +51,28 @@
     private static final String INTERCEPTED_URI = "intercepted_uri";
     private static final String INTERCEPTOR_CLASSNAME = "interceptor_classname";
 
+    public static void syncAnalyticsOptionWithWpCom(Context ctx,
```

Comment:
This method seems misnamed. In addition to syncing the setting with wp.com, it also updates the local account and sets the shared preference.